.github/workflows/initial_CI.yml

```diff
@@ -0,0 +1,40 @@
+name: Django CI
+
+on:
+#   push:
+#     branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 20
```

Comment:
Is that needed?

Reply:
@lioramilbaum yes. it's failed without it.

Reply:
strange. According to the documentation [1] it should used only when using matrix job strategy. I don't you are using it now. Maybe you have used it before the latest changes.

Reply:
fixed